src/lib/homestore_backend/hs_pg_manager.cpp

```diff
@@ -127,12 +136,19 @@ void HSHomeObject::on_create_pg_message_commit(int64_t lsn, sisl::blob const& he
         return;
     }
 
+    // select chunks for pg
+    const auto chunk_size = chunk_selector()->get_chunk_size();
+    const chunk_num_t num_chunk = sisl::round_up(pg_info.size, chunk_size) / chunk_size;
+    LOGINFOMOD(homeobject, "[Hooper] on create commit create pg {}, num_chunk {}, pg_info_size {}", pg_id, num_chunk, pg_info.size);
+
+    chunk_selector()->select_chunks_for_pg(pg_id, num_chunk);
```

Comment:
suggesting failing here. I think before L160 we can  *fail* a pg creation by not adding to map, following data write will fail as chunk_selector cannot give back hint,  append_log will fail

c.f https://github.com/eBay/HomeStore/blob/master/src/lib/replication/repl_dev/common.cpp#L91-L92

Reply:
Done